src/lib.rs

```diff
@@ -241,21 +248,26 @@ mod hwb;
 
 mod hues;
 
-mod tristimulus;
 mod convert;
 mod equality;
+pub mod chromatic_adaptation;
+pub mod white_point;
+pub mod matrix;
+pub mod rgb_primaries;
```

Comment:
I guess `matrix` can be public, but the state of `rgb_primaries` is a bit too fluid.